docs/_includes/donut-chart.html

```diff
@@ -0,0 +1,156 @@
+<div id="donut-chart">
+    {% include section-header.html name="donut-chart" version=page.versions.donut-chart %}
+    <p>The donut chart is used to show the breakdown of a single part-to-whole relationship within a total.</p>
+    <h3>Default Chart Legend</h3>
```

Comment:
The headings and copy look to be related to the legend. Maybe it was a copy/paste issue? They should be referencing the donut chart.

Reply:
totally a copy paste issue. fixing it up